config/routes/placements.rb

```diff
@@ -1,3 +1,7 @@
-scope module: :placements, as: :placements, constraints: { host: ENV["PLACEMENTS_HOST"] } do
+scope module: :placements,
+      as: :placements,
+      constraints: {
+        host: ENV["PLACEMENTS_HOST"]
+      } do
```

Comment:
We should avoid adding any changes that aren't relevant to this specific PR objective.